### PR TITLE
testutil/unticketedpr: add no-issue PRs to github project board

### PR DESCRIPTION
**Pull request by xenowits**
Add PRs with no associated ticket to github project board.

category: misc
ticket: #987

### CODE REVIEW

File: testutil/unticketedpr/unticketedpr.go [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+// more details.
+//
+// You should have received a copy of the GNU General Public License along with
+// this program.  If not, see <http://www.gnu.org/licenses/>.
+
+// The unticketedpr command verifies if a ticket with a "none" tag is present in the closed PR. If such a PR exists,
+// the command sets NONE_TICKET_PRESENT to true.
+//
+//nolint:wrapcheck,revive,cyclop,forbidigo
+package main
```
Comment (corverroos) [inline testutil/unticketedpr/unticketedpr.go:20]: main package comment start with `Command unticketedpr does blah blah`

File: .github/workflows/unticketed-pr.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "1.18.0"
+
+      - name: Verify PR
+        run: go run testutil/verifypr/verifypr.go
+
+      - name: Set env.NONE_TICKET_PRESENT
+        run: go run testutil/unticketedpr/unticketedpr.go
```
Comment (corverroos) [inline .github/workflows/unticketed-pr.yml:21]: suggest rename: `trackpr`

File: .github/workflows/unticketed-pr.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    env:
+      GITHUB_PR: ${{ toJSON(github.event.pull_request) }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "1.18.0"
+
+      - name: Verify PR
+        run: go run testutil/verifypr/verifypr.go
```
Comment (corverroos) [inline .github/workflows/unticketed-pr.yml:18]: why verify again?

File: .github/workflows/unticketed-pr.yml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+      - name: Set env.NONE_TICKET_PRESENT
+        run: go run testutil/unticketedpr/unticketedpr.go
+
+      - name: Add PR To Project
+        if: ${{ success() && env.NONE_TICKET_PRESENT }}
```
Comment (corverroos) [inline .github/workflows/unticketed-pr.yml:24]: see issue, but you need to add size and sprint and status, not sure this action supports that, you need to manually add it to the project from `trackpr`

File: testutil/unticketedpr/unticketedpr.go [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+	return nil
+}
+
+// noneTicket returns true if the ticket is "none" and returns false otherwise.
+// It doesn't verify the body and assumes verifyPR step has already succeeded.
+func containsNoneTicket(body, noneTicket string) bool {
+	for _, line := range strings.Split(body, "\n") {
+		const ticketTag = "ticket:"
+		if strings.HasPrefix(line, ticketTag) {
```
Comment (corverroos) [inline testutil/unticketedpr/unticketedpr.go:90]: avoid nesting

File: .github/workflows/unticketed-pr.yml (hunk 4 above)
Comment (xenowits) [reply .github/workflows/unticketed-pr.yml]: yeah, this github action is pretty basic and just adds to project board, can't assign size, sprint etc. will do it manually then

**Pull request closed without merging**